modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3149,6 +3149,21 @@ private boolean _hasInsertValues(
 		return true;
 	}
 
+	private boolean _hasUpdateValues(ObjectField objectField) {
```

Comment:
Can we put this inside `ObjectFieldImpl`?

Reply:
Done!